.github/workflows/ci.yml

```diff
@@ -91,7 +91,7 @@ jobs:
 
     runs-on: ubuntu-20.04
 
-    needs: [php-lint, composer-validate]
+    needs: [ php-lint, composer-validate ]
```

Comment:
Why spaces immediately inside the braces?  We don't have this formatting in PHP and JSON, and wouldn't have it in JavaScript (if we had any).  It seems inconsistent to me, and I don't see a good reason for it.

Reply:
That's the default setting of the PhpStorm YAML autoformatter. (I'd like contributors to be able to use the PhpStorm default settings as much as possible when they work on Emogrifier so it's as small as a hassle for them as possible.)

Reply:
OK.